### ISSUE TITLE
Checkbox seems broken

### ISSUE THREAD

**Issue by ghost** [vnotex/vnote#1131]
OS Version (操作系统): macOS

VNote Version (Help - About VNote - Version: )(VNote版本): 2.8.2
Symptoms (表现): Task checkbox seems broken

How to Repro (如何复现): 
When you add a task list as below
```
- [x] this is a complete item.
- [ ] this is an incomplete item.
```
 then When I try to add a check mark for incomplete item in read/preview mode, checkbox doesn't support.

**Comment by tamlok** [comment]
Yes, it works as expected. We need to consider if it is appropriate to modify a note in read mode by clicking the checkbox.

**Comment by eblanshey** [comment]
This would be a very nice feature! Also would be nice to add a checkbox shortcut in the tool bar.

**Comment by sapphire008** [comment]
Agreed, it would be a nice feature to be allow checkbox clicking / ticking in read mode.

**Comment by tamlok** [comment]
Supported now
